Library/Equipment/Dungeon Fantasy RPG.eqp

```diff
@@ -10914,17 +10932,7 @@
 		</ranged_weapon>
 		<categories>
 			<category>Missile Weapon</category>
-		</categories>
-	</equipment>
-	<equipment version="4">
-		<quantity>1</quantity>
-		<description>Prodd Pellet</description>
-		<legality_class>4</legality_class>
-		<value>0.1</value>
-		<weight>0.06 lb</weight>
-		<reference>DFA104</reference>
-		<categories>
-			<category>Ammunition</category>
+			<category>UsesAmmoType:Prodd</category>
```

Comment:
Why did you remove the Prodd Pellet?

Reply:
It is a duplicate of "Lead Pellet (Prodd)".

Reply:
Indeed! Good catch.